ballot/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import patterns, url
+from django.contrib import admin
+
+from . import views
+
+admin.autodiscover()
+
+urlpatterns = patterns(
+    '',
+    url(r'^measure/(?P<measure_id>[0-9]+)$', views.measure_view),
+    url(r'^ballot/$', views.ballot_view))
```

Comment:
> I see. I didn't understand from the frontend code the full URL that would be sent to the backend. Is there a good way to eyeball that?

Oops, no that's not what I meant. I misunderstood what you mean by "mock-up url". When I was creating the frontend urls, I was mirroring what I thought the API should be. I put no information with the stubs about what the API would look like. You can use the frontend URLs as a rule of thumb. Obviously we've worked on the vocabulary on the backend, but haven't updated the frontend yet.

You can click through the pages and use the Demo links to get you to the main entities.

Also, in a single state.js file, the states are nested, so you can append the urls to the base states to figure out what the full url would be. e.g.
appMain.measure, /measure/:measure_id
appMain.measure.supporting, /supporting

so the full route for measure supporters would be /measure/:measure_id/supporting.